prowler/providers/aws/services/elasticache/elasticache_service.py

```diff
@@ -30,6 +30,9 @@ def _describe_cache_clusters(self, regional_client):
                     if not self.audit_resources or (
                         is_resource_filtered(cluster_arn, self.audit_resources)
                     ):
+                        version = cache_cluster.get("EngineVersion", "0.0")
+                        version = float(version[:3])
+
```

Comment:
```suggestion
                        version = version.parse(cache_cluster.get("EngineVersion", "0.0"))
```

Please, use version function from the packaging library.

Reply:
`from packaging import version`